src/main/java/uk/nhs/digital/nhsconnect/lab/results/utils/JmsHeaders.java

```diff
@@ -1,5 +1,8 @@
 package uk.nhs.digital.nhsconnect.lab.results.utils;
 
 public final class JmsHeaders {
+
+    public static String OPERATION_ID = "OperationId";
+    public static String TRANSACTION_TYPE = "TransactionType";
```

Comment:
Sailesh is making `CONVERSATION_ID` final in #12 (going into #11), and these will need to be final too

Reply:
Done - i must have missed this